common/db.go

```diff
@@ -217,6 +217,11 @@ func InitDB(dbPath string) (*DB, error) {
 		ELSE excluded.deactivationRound END 
 	`)
 	d.updateOrch = stmt
+	if err != nil {
```

Comment:
Check error immediately after statement preparation to follow the same convention as the rest of the statement preparations?

Reply:
fixed